src/Exception/SyntaxErrorException.php

```diff
@@ -4,7 +4,7 @@
 
 class SyntaxErrorException extends \Exception
 {
-    public function __construct($original, $value, $position)
+    public function __construct(string $original, string $value, ?int $position)
```

Comment:
Ok for `?int` but a cast is needed in `Type/Parser/Parser.php:L101`

Reply:
👍